Create add-unanswered-to-project.yml

nil-is-all opened the pull request:
Adding a workflow automation to dynamically add unanswered PRs and issues to project view (https://github.com/orgs/pytorch/projects/136)

nil-is-all committed:
Create add-unanswered-to-project.yml

Adding a workflow automation to dynamically add unanswered PRs and issues to project view

nil-is-all committed:
Updated to new project board

GregoryComer commented on .github/workflows/add-unanswered-to-project.yml:
```diff
@@ -0,0 +1,76 @@
+                  draft: false,
+                }
+              );
+              for (const pr of prs) {
```
Can you remove the unapproved constraint? I think it might be good to track approved PRs that haven't been merged yet.

GregoryComer commented on .github/workflows/add-unanswered-to-project.yml:
```diff
@@ -0,0 +1,76 @@
+            }
+
+            try {
+              // Add open issues (not PRs)
```
It looks like it's only adding issues. Can we add both PRs and issues?

GregoryComer commented on .github/workflows/add-unanswered-to-project.yml:
```diff
@@ -0,0 +1,76 @@
+name: Add Unanswered PRs and Issues to PyTorch Org Project 136
```
What do you think about renaming to "Add External PRs and Issues...", since we're not filtering by unanswered yet?

nil-is-all committed:
Updated title, includes approved PRs

1) Changes title from "Add Unanswered PRs and Issues to PyTorch Org Project 136" to "Add Open External Contributor PRs and Issues to PyTorch Org Project 136"

2) Changed constraint to add PRs from adding approved PRs as well

nil-is-all committed:
Merge branch 'main' into patch-1
